build/run.js

```diff
@@ -482,6 +482,7 @@ async function updateBuildVersion (argv) {
     if (config.buildVersion !== commitHash || config.target !== target){
         config.target = target
         config.buildVersion = commitHash
+        config.args = argv
```

Comment:
Why we need this? I don't see any usage in this PR.

Reply:
Removed.